x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/index.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export { SearchIndexRouter } from './search_index_router';
+export { SearchIndex } from './search_index';
```

Comment:
suggestion: remove this index.ts file as it's not really adding much, right?

Reply:
Will circle back to this, I'm gonna merge to not deal with another round of CI

Reply:
lmao nvm, tests failing, I will remove